modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+
+[id="osdk-upgrading-v130-to-v180_{context}"]
+= Upgrading projects for Operator SDK {osdk_ver}
+
+The following upgrade steps must be performed to upgrade an existing Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+. Make the following changes to your `PROJECT` file:
+
+.. Update the `PROJECT` file `plugins` object to use `manifests` and `scorecard` objects.
++
+The `manifests` and `scorecard` plug-ins that create Operator Lifecycle Manager (OLM) and scorecard manifests now have plug-in objects that direct `create` subcommands to create related files.
+
+** For Go-based Operator projects, an existing Go-based plug-in configuration object is already present. While the old configuration is still supported, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
++
+.Old configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+----
++
+.New configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+** Optional: For Ansible- and Helm-based Operator projects, the plug-in configuration object previously did not exist. While not required to add, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
++
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+.. The `PROJECT` config version `3-alpha` must be upgraded to `3`. The `version` key in your `PROJECT` file represents the `PROJECT` config version:
++
+.Old `PROJECT` file
+[source,yaml]
+----
+version: 3-alpha
+resources:
+- crdVersion: v1
+...
+----
++
+Version `3-alpha` has been stabilized as link:https://book.kubebuilder.io/migration/v2vsv3.html[version 3] and contains a set of config fields sufficient to fully describe a project. While this change is not technically breaking because the spec at that version was alpha, it was used by default in `operator-sdk` commands, so it should be marked as breaking and have a convenient upgrade path.
+
+... Run the `alpha config-3alpha-to-3` command to convert most of your `PROJECT` file from version `3-alpha` to `3` and output comments with directions where automatic conversion is not possible:
```

Comment:
Sharing a thought: Could the "...and output comments..." part be a separate sentence that follows the example output?  It's a lot of instructional text--I have no idea if the comments are often printed or rarely printed.

Reply:
This felt like a lot to jam in at once to me as well. Will split it up.